simpleai_base/models_info.py

```
@@ -676,13 +677,22 @@ def get_model_filepath(self, catalog='', model_path='', muid=None):
+    def get_model_names(self, catalog, filters=None, casesensitive=False, reverse=False, user=None):
+        if filters is None:
+            filters = []
         result = []
         result_reverse = []
         for f in self.m_info.keys():
             cata = f.split('/')[0]
             m_path_or_file = f[len(cata) + 1:].replace('/', os.sep)
             if catalog and cata == catalog:
+                if catalog =="loras":
+                    model_type = m_path_or_file.split(os.sep)[0]
+                    new = m_path_or_file[len(model_type) + 1:].replace('/', os.sep)
```

Ruff (0.11.9)

Local variable `new` is assigned to but never used

Remove assignment to unused variable `new`

(F841)